modules/mbedtls/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ if(CONFIG_MBEDTLS_BUILTIN)
 
   target_include_directories(mbedTLS INTERFACE
     ${ZEPHYR_CURRENT_MODULE_DIR}/include
+    ${ZEPHYR_CURRENT_MODULE_DIR}/library
```

Comment:
I guess this is because `psa_crypto_cipher.h` is included incorrectly: 
https://github.com/zephyrproject-rtos/mbedtls/blob/7fed49c9b9f983ad6416986661ef637459723bcb/library/psa_crypto_cipher.c#L25

I guess instead of adding `library` as include directory, it should be fixed in mbedtls repo. At least this is an internal header which obviously should be included with double quotes and in 2 other places where this file is included, it is included with double quotes.

Reply:
I suggest to leave it `as is` in PR and to create a separate task to solve this in the right manner. It will require at least 2 PRs with west mbtle link update.

Reply:
https://github.com/zephyrproject-rtos/mbedtls/pull/37   PR in mbedtls